__tests__/git-operations-test.ts

```diff
@@ -0,0 +1,249 @@
+    await git('branch -m main');
+    await git('checkout -b branch-1');
+
+    applyTodoChanges(
+      todoStorageFilePath,
+      new Set(),
+      buildTodosFromOperations([
+        'remove|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js',
+        'remove|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js',
+      ])
+    );
+
+    expect(readTodoStorageFileContents(todoStorageFilePath)).toMatchInlineSnapshot(`
+      "add|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      add|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      remove|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      remove|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      "
+    `);
+
+    await git('add -A');
+    await git(['commit', '-m', '"Add removes"']);
+    await git(`checkout main`);
+
+    expect(readTodoStorageFileContents(todoStorageFilePath)).toMatchInlineSnapshot(`
+      "add|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      add|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      "
+    `);
+
+    await git('merge branch-1');
+
+    expect(readTodoStorageFileContents(todoStorageFilePath)).toMatchInlineSnapshot(`
+      "add|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      add|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      remove|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      remove|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js
+      "
+    `);
+  });
+
+  it('can auto-resolve conflicts in git storage file via merge', async () => {
+    const todoStorageFilePath = getTodoStorageFilePath(tmp);
+
+    applyTodoChanges(
+      todoStorageFilePath,
+      buildTodosFromOperations([
+        'add|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js',
+        'add|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js',
+      ]),
+      new Set()
+    );
+
+    await git('init');
+    await setGitUser();
```

Comment:
Can you move this up into the beforeEach

Reply:
Done.